### ISSUE TITLE
2.11.x update flags Japanese characters on Linux

### ISSUE THREAD

**Issue by mattcone** [jdkato/txtlint#371]
Here there — I've run into a strange problem after upgrading to 2.11.x. 

For years now, we've been successfully using Vale to check all of our documentation in both English and Japanese. The spelling test ignored the Japanese characters, as documented. After upgrading to version 2.11.x, Vale still ignores those characters when run on macOS. But when version 2.11.x is run on TravisCI, the spelling test flags all of the Japanese characters. We didn't change anything in the configuration files, so we're not sure what could be causing this other than the update to 2.11.x.

### Spelling.yml

```yaml
extends: spelling
message: "Did you really mean '%s'?"
level: error
ignore: .vale/dictionary.txt
```

### macOS output

```bash
lobo:fastly-docs mcone$ vale --v
vale version 2.11.1
lobo:fastly-docs mcone$ rake vale
Testing vale structure... Ok!
Testing vale command... Ok!
./bin/vale collections/_changelog collections/_changelog_archive collections/_changelog_significant collections/_en_guides collections/_ja_guides collections/_en_guides_primary collections/_en_guides_secondary collections/_products collections/_products_primary collections/_waf_rule_sets
✔ 0 errors, 0 warnings and 0 suggestions in 792 files.
```

### Install script for TravisCI

```bash
#!/bin/bash

curl -sfL https://install.goreleaser.com/github.com/ValeLint/vale.sh | sh -s v2.11.1
export PATH="./bin:$PATH"
vale -v
```

### Excerpt from TravisCI output with Vale 2.11.x

```
script/install_vale
ValeLint/vale info checking GitHub for tag 'v2.11.1'
ValeLint/vale info found version: 2.11.1 for v2.11.1/Linux/64-bit
ValeLint/vale info installed ./bin/vale
vale version 2.11.1
Testing vale structure... Ok!
Testing vale command... Ok!
./bin/vale collections/_changelog collections/_changelog_archive collections/_changelog_significant collections/_en_guides collections/_ja_guides collections/_en_guides_primary collections/_en_guides_secondary collections/_products collections/_products_primary collections/_waf_rule_sets

collections/_changelog/2020-04-01-changes.md
62:25  error  Did you really mean '日本語'?   Fastly.Spelling 
64:4   error  Did you really mean             Fastly.Spelling 
            'よくある'?                                     
64:13  error  Did you really mean 'エラー'?   Fastly.Spelling 
65:4   error  Did you really mean '無料'?     Fastly.Spelling 
65:11  error  Did you really mean 'の設定'?   Fastly.Spelling 
66:8   error  Did you really mean             Fastly.Spelling 
            'オプション'?   
```

### Excerpt from TravisCI output with Vale 2.10.7

```
script/install_vale
ValeLint/vale info checking GitHub for tag 'v2.10.7'
ValeLint/vale info found version: 2.10.7 for v2.10.7/Linux/64-bit
ValeLint/vale info installed ./bin/vale
vale version 2.10.7
Testing vale structure... Ok!
Testing vale command... Ok!
./bin/vale collections/_changelog collections/_changelog_archive collections/_changelog_significant collections/_en_guides collections/_ja_guides collections/_en_guides_primary collections/_en_guides_secondary collections/_products collections/_products_primary collections/_waf_rule_sets
✔ 0 errors, 0 warnings and 0 suggestions in 790 files.
```

**Comment by jdkato** [comment]
Would it be possible to share the file (or part of it)?

**Comment by mattcone** [comment]
Sure, no problem: https://gist.github.com/mattcone/b33719badcb58f62ac4f6b5399679d10

I'm not sure if I conveyed this, but I believe *all* Japanese characters are being flagged in all of our files. The Travis build log with Vale 2.11.x must be thousands of lines long.

**Comment by jdkato** [comment]
This should be fixed in the next release.

That said, I'm not sure why you only experienced it on Linux -- I was able to reproduce on it both Linux and macOS, and there shouldn't be anything Linux-specific about how `spelling` works.

**Comment by mattcone** [comment]
Thank you! Much appreciated!

**Comment by jdkato** [comment]
The new release is [out now](https://github.com/errata-ai/vale/releases/tag/v2.11.2).

Also note that, in light of the GoDownloader project being deprecated yesterday, I no longer reccomend that you use the `https://install.goreleaser.com/github.com/ValeLint/vale.sh` script (I'm unsure if the hosting will remain active).
